### PR TITLE
Update contributions

### PR DESCRIPTION
**Pull request by jpc20**
# Pull request checklist

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Current behavior

- Previously, minContribution was the minimum that could be donated to a campaign and everyone that donated became an approver.

## New behavior

- Now the minContribution represents the minimum threshold for a contributor to become an approver of requests.

### CODE REVIEW

File: ethereum/contracts/Campaign.sol [hunk 1]
```diff
@@ -28,9 +28,7 @@ contract Campaign {
     }
 
     function contribute() public payable {
-        require(msg.value > minimumContribution);
-
-        if (approvers[msg.sender] != true) {
+        if (approvers[msg.sender] != true && msg.value > minimumContribution) {
```
Comment (jpc20) [inline ethereum/contracts/Campaign.sol:31]: I think this is all that needed to be changed in the contract... much easier than I thought.

File: src/components/CampaignDetails.vue [hunk 2]
```diff
@@ -13,15 +13,15 @@
...
         <button :disabled="!contribution"
         @click="submitContribution">
         Submit Contribution </button>
-        <p class="user-message"
-          v-if="userMessage">
-          Please enter a contribution over {{campaign.min_contribution}}!
-        </p>
+        <button :disabled="!contribution"
+        @click="convertToDollars">
+        Convert To USD </button>
```
Comment (jpc20) [inline src/components/CampaignDetails.vue:24]: This button doesn't do anything yet but thought I would add it to make sure we don't forget.

File: src/components/CampaignDetails.vue [hunk 3]
```diff
@@ -49,19 +50,28 @@ export default {
...
+        try {
+          await axios.put(
+            `${VUE_APP_API_URL}campaigns/${this.campaign.id}`,
+            { contributors: (this.campaign.contributors + 1) }
+          );
+        } catch (error) {
+          return { error };
+        }
+      }
+    },
```
Comment (jpc20) [inline src/components/CampaignDetails.vue:71]: Feel free to refactor or request changes on this, still not totally comfortable making changes on the FE

Comment (edmdc) [reply src/components/CampaignDetails.vue:71]: This looks pretty good to me, I'm a little unsure of how JS would make that string comparison on line 61.

**Review by attom2 — APPROVED** [review]

**Pull request merged**